CHANGELOG.md

```diff
@@ -54,6 +54,7 @@ Our versioning strategy is as follows:
 * `[sitecore-jss-nextjs]` Fix middleware redirect when the target use regexp with querystring ([#1449](https://github.com/Sitecore/jss/pull/1449)) ([#1460](https://github.com/Sitecore/jss/pull/1460))
 * `[templates/nextjs]` Fix incorrectly named .gitignore file `\scripts\temp\.npmignore` ([#1463](https://github.com/Sitecore/jss/pull/1463))
 * `[angular]` Avoid sending two dictionary service calls when switching language and refreshing the page ([#1473](https://github.com/Sitecore/jss/pull/1473))
+* Fix installed sitecore-jss-* dependency version ([#1478](https://github.com/Sitecore/jss/pull/1478))
```

Comment:
Maybe we should provide a bit more context to the issue here for customers.
```suggestion
* Fix `@sitecore-jss/sitecore-jss-*` dependency versions (now uses tilde `~` instead of caret `^`). Required since our change in versioning strategy with JSS 21.1.x where a minor version may include breaking changes in framework packages. ([#1478](https://github.com/Sitecore/jss/pull/1478))
```